app/src/main/java/umc/everyones/lck/presentation/login/SignupSuccessFragment.kt

```diff
@@ -14,52 +17,58 @@ import umc.everyones.lck.presentation.base.BaseFragment
 import umc.everyones.lck.util.NicknameManager
 import umc.everyones.lck.util.TeamData
 import androidx.navigation.fragment.navArgs
+import kotlinx.coroutines.launch
+import umc.everyones.lck.User
 
 @AndroidEntryPoint
 class SignupSuccessFragment : BaseFragment<FragmentSignupSuccessBinding>(R.layout.fragment_signup_success) {
 
-    private val args: SignupSuccessFragmentArgs by navArgs()
+    private val viewModel: SignupViewModel by viewModels()
+    private lateinit var nicknameManager: NicknameManager
+    private val nickname: String by lazy {
+        arguments?.getString("nickname") ?: ""
+    }
 
     override fun initObserver() {
-        // No observers needed here
+        // No observers needed for this fragment
     }
 
     override fun initView() {
-        val nicknameManager = NicknameManager(requireContext())
+        nicknameManager = NicknameManager(requireContext())
 
-        // Retrieve arguments using Safe Args
-        val selectedTeam = args.selectedTeam
-        val nickname = args.nickname
-        val profileImageUri = args.profileImageUri
+        // Fetch user details from ViewModel
+        lifecycleScope.launch {
+            val user = viewModel.getUser(nickname)
+            if (user != null) {
+                displayUserInfo(user)
+            }
+        }
 
-        // Save nickname
-        nicknameManager.addNickname(nickname)
+        // Handle Next button click
+        binding.ivSignupSuccessNext.setOnClickListener {
+            val intent = Intent(requireContext(), MainActivity::class.java)
+            startActivity(intent)
+            requireActivity().finish()
```

Comment:
```suggestion
            Intent(requireContext(), MainActivity::class.java).apply { 
                startActivity(this)
            }
            requireActivity().finish()
```
apply 확장함수를 사용하면 불필요한 intent 변수를 생성하지 않고 사용 가능합니다 참고해주세요~~